nixos/doc/manual/release-notes/rl-2111.section.md

```diff
@@ -111,6 +111,8 @@ In addition to numerous new and upgraded packages, this release has the followin
 
   * The data directory remains located at `/var/lib/bitwarden_rs`, for backwards compatibility.
 
+* Python package `pytest-rerunfailures` was upgraded to a new major release. See [upstream changelog](https://github.com/pytest-dev/pytest-rerunfailures/blob/10.1/CHANGES.rst#101-2021-07-02).
+
```

Comment:
```suggestion
```

I don't think we need this. No one really cares that much about this package.

Reply:
I thought so, see https://github.com/NixOS/nixpkgs/pull/129179#discussion_r663414188.

Removed the release notes entries by [going back to 77a197f54a1e6e2c2024b8a3a6c601238dce6d22](https://github.com/NixOS/nixpkgs/compare/47cd2dc73941a383bebdd2438e6a5bc6fdecaa72..77a197f54a1e6e2c2024b8a3a6c601238dce6d22) (from before I amended the addition of the release notes entry).